lua/oil/config.lua

```diff
@@ -332,6 +334,7 @@ local M = {}
 ---@field max_height integer
 ---@field border string|string[]
 ---@field win_options table<string, any>
+---@field get_win_title fun(path: string): string
```

Comment:
You'll also need to add an entry on line 347 below. This is for the internal typed representation of the config, below is for the type that is passed in to `oil.setup()`, so is the same except everything is optionally nil.

Reply:
Thanks, adjusted.